bentoml/utils/cloudpickle.py

```diff
@@ -844,7 +844,7 @@ def save_global(self, obj, name=None, pack=struct.pack):
         elif obj is type(NotImplemented):
             return self.save_reduce(type, (NotImplemented,), obj=obj)
 
-        if obj.__module__ == "__main__":
+        if hasattr(obj, "__module__") obj.__module__ == "__main__":
```

Comment:
```suggestion
        if hasattr(obj, "__module__") and obj.__module__ == "__main__":
```

Reply:
@bojiang just changed. Sorry for that mistake